src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+    Component,
+    Element,
+    h,
+    Event,
+    EventEmitter,
+    Prop,
+} from '@stencil/core';
+import { BreadcrumbsItem } from './breadcrumbs.types';
+import {
+    makeEnterClickable,
+    removeEnterClickable,
+} from '../../util/make-enter-clickable';
+import { createRandomString } from '../../util/random-string';
+
+/**
+ * A Breadcrumb consists of a list of distinct "places" that a user has gone through,
+ * before ending up where they are right now, in a website or an application.
+ *
+ * These "places" can be for example _pages_ of a website, which are hierarchically
+ * laid out before the current page that the user is looking at.
+ * They could also be _steps_ which the user has gone through, which perhaps have no
+ * hierarchical relation with each other, but has eventually led the user "here".
+ *
+ * :::note
+ * - Where the user currently is, is always the last step of the breadcrumb.
+ * - A breadcrumbs never shows where users can go after this place.
+ * It only illustrates where user has been before ending up here.
+ * If the path that a user can take is not changing and if next steps are clear,
+ * you can use the [Progress flow component](#/component/limel-progress-flow) instead.
+ * :::
+ *
+ * Breadcrumbs are often placed horizontally before the main content of the current screen.
+ * @exampleComponent limel-example-breadcrumbs-links
+ * @exampleComponent limel-example-breadcrumbs-buttons
+ * @exampleComponent limel-example-breadcrumbs-icons
+ * @exampleComponent limel-example-breadcrumbs-divider
+ * @exampleComponent limel-example-breadcrumbs-icon-color
+ * @exampleComponent limel-example-breadcrumbs-styling
+ */
+
+@Component({
+    tag: 'limel-breadcrumbs',
+    shadow: true,
+    styleUrl: 'breadcrumbs.scss',
+})
+export class Breadcrumbs {
+    /**
+     * List of items in the breadcrumbs,
+     * each representing a step or a page.
+     */
+    @Prop()
+    public items: BreadcrumbsItem[];
+
+    /**
+     * The visual divider that separates items.
+     * It must be a single character such as `-` or `,`.
+     */
+    @Prop()
+    public divider: string = '›';
+
+    /**
+     * Fired when clicking on buttons (not links!)
+     * inside the breadcrumbs.
+     */
+    @Event()
+    public select: EventEmitter<BreadcrumbsItem>;
+
+    @Element()
+    private host: HTMLLimelBreadcrumbsElement;
+
+    private button: HTMLButtonElement;
+    private anchor: HTMLAnchorElement;
+
+    public render() {
+        return (
+            <ol
+                role="navigation"
+                aria-label="Breadcrumb"
+                style={{ '--limel-breadcrumbs-divider': `'${this.divider}'` }}
+            >
+                {this.renderSteps(this.items)}
+                {this.renderLastStep(this.items)}
+            </ol>
+        );
+    }
+
+    public componentWillRender() {
+        // Avoid polluting the "clickable registry" with unreferenced nodes
+        this.removeEnterClickable();
+    }
```

Comment:
Is this by mistake? I don't think we need it

Reply:
this is needed when we want to prevent the <kbd>Enter</kbd> to trigger the action too many times. (when the key is pressed down and held)